### PR TITLE
Readme rework for 0.4 release

### PR DESCRIPTION
**Pull request by markurtz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,31 +14,30 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# ![icon for DeepSparse](https://raw.githubusercontent.com/neuralmagic/deepsparse/main/docs/source/icon-deepsparse.png) DeepSparse Engine
+<h1><img src="https://raw.githubusercontent.com/neuralmagic/deepsparse/main/docs/source/icon-deepsparse.png" />&nbsp;&nbsp;DeepSparse</h1>
```
Comment (jeanniefinks) [inline README.md:17]:
```suggestion
<h1><img alt="tool icon" src="https://raw.githubusercontent.com/neuralmagic/deepsparse/main/docs/source/icon-deepsparse.png" />&nbsp;&nbsp;DeepSparse</h1>
```

File: README.md [hunk 2]
```diff
@@ -51,37 +50,67 @@ limitations under the License.
...
-This repository includes package APIs along with examples to quickly get started benchmarking and inferencing sparse models. 
+The [GitHub repository](https://github.com/neuralmagic/deepsparse) includes package APIs along with examples to quickly get started benchmarking and inferencing sparse models.
+
+<img src="https://docs.neuralmagic.com/docs/source/infographics/deepsparse.png" width="960px" />
+
+## Highlights
+
+<p>
+    <a href="https://neuralmagic.com/blog/benchmark-resnet50-with-deepsparse/">
+        <img src="https://docs.neuralmagic.com/docs/source/highlights/deepsparse/resnet-50.png" width="256px" />
```
Comment (jeanniefinks) [inline README.md:64]:
```suggestion
        <img alt="ORT: 296 images/sec vs DeepSparse: 2305 images/sec on 24 cores" src="https://docs.neuralmagic.com/docs/source/highlights/deepsparse/resnet-50.png" width="256px" />
```

File: README.md [hunk 3]
```diff
@@ -51,37 +50,67 @@ limitations under the License.
...
+    </a>
+    <a href="https://neuralmagic.com/blog/benchmark-yolov3-on-cpus-with-deepsparse/">
+        <img src="https://docs.neuralmagic.com/docs/source/highlights/deepsparse/yolov3.png" width="256px" />
```
Comment (jeanniefinks) [inline README.md:67]:
```suggestion
        <img alt="YOLOv3, b64 - PyTorch: 6.9 images/sec vs. DeepSparse: 46.5 images/sec" src="https://docs.neuralmagic.com/docs/source/highlights/deepsparse/yolov3.png" width="256px" />
```

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
minor corrections

**Review by jeanniefinks — APPROVED** [review]

**Review by NMsasa — APPROVED** [review]

**Pull request merged**